src/iframe/IframeArticlePage.js

```diff
@@ -21,8 +21,26 @@ import { getAllDimensions } from '../util/trackingUtil';
 import PostResizeMessage from './PostResizeMessage';
 import FixDialogPosition from './FixDialogPosition';
 import { SocialMediaMetadata } from '../components/SocialMediaMetadata';
+import { fetchResource } from '../containers/Resources/resourceApi';
+import config from '../config';
 
 class IframeArticlePage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      contentUrl: '',
+    };
+  }
+
+  componentDidMount() {
+    const resourceId = this.props.location.pathname.split('/')[3];
+    fetchResource(resourceId).then(resource => {
+      this.setState({
+        contentUrl: resource.path,
```

Comment:
Dersom path ikkje finnes for ressursen må vi la være å vise denne lenka.

Reply:
Det er sant. Fikset det nå